### ISSUE TITLE
No response to follow up question

### ISSUE THREAD

**Issue by khanhlnq** [Zerg00s/open-ai-teams-chat-bot#2]
Hi @Zerg00s , I tested and got response for my first message. But when I reply, the flow didn't give response. Could we update the flow to support follow-up question?

![image](https://github.com/Zerg00s/open-ai-teams-chat-bot/assets/353620/0baf8502-d3d4-4879-9e3b-de427dfa21f8)

**Comment by Zerg00s** [comment]
good idea, @khanhlnq . I can't promise I can do it quickly, but I will see what I can do in a few days. it should be possible.
